modules/porous_flow/src/userobjects/PorousFlowCapillaryPressure.C

```diff
@@ -0,0 +1,35 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "PorousFlowCapillaryPressure.h"
+
+template <>
+InputParameters
+validParams<PorousFlowCapillaryPressure>()
+{
+  InputParameters params = validParams<GeneralUserObject>();
+  params.addRangeCheckedParam<Real>(
+      "sat_lr",
+      0.0,
+      "sat_lr >= 0 & sat_lr <= 1",
+      "Liquid residual saturation.  Must be between 0 and 1. Default is 0");
```

Comment:
Aha, i get it.  i've never put sat_lr = residual_from_relperm: i've always used sat_lr = 0.  Similarly in my old COSFLOW code we had an "immobile saturation" (which is the residual saturation in the relperm) and a "residual saturation" (which is this sat_lr) and we ran lots of rather large models with them both zero.   i wonder whether we would have gained better convergence if we used TOUGH's convention.

When they say "slightly smaller", do they mean sat_lr=0.999*residual_in_relperm, or sat_lr=0.1*residual_in_relperm, or something else?   What do you usually do?   I'd really like to speed up models by changing these parameters, considering that they're never actually measured accurately (for entire rock masses, especially!)